atlasdb-conjure/src/main/java/com/palantir/atlasdb/http/v2/ConjureJavaRuntimeTargetFactory.java

```diff
@@ -60,7 +59,10 @@ private ConjureJavaRuntimeTargetFactory() {
                 Optional.empty(),
                 trustContext.orElseThrow(() -> new IllegalStateException("CJR requires a trust context")));
         T client = JaxRsClient.create(
-                type, addAtlasDbRemotingAgent(parameters.userAgent()), HOST_METRICS_REGISTRY, clientConfiguration);
+                type,
+                addAtlasDbRemotingAgent(parameters.userAgent()),
+                NoOpHostEventsSink.INSTANCE,
```

Comment:
My understanding is that this is fine because we never actually retrieve the metrics from `HOST_METRICS_REGISTRY`, correct?

Reply:
Correct